code/modules/reagents/newchem/medicine.dm

```diff
@@ -819,13 +817,88 @@ datum/reagent/stimulants/reagent_deleted(var/mob/living/M as mob)
 datum/reagent/insulin
 	name = "Insulin"
 	id = "insulin"
-	description = "Increases sugar depletion rates."
+	description = "A hormone generated by the pancreas responsible for metabolizing carbohydrates and fat in the bloodstream."
 	reagent_state = LIQUID
 	color = "#C8A5DC"
 datum/reagent/insulin/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
-	if(M.sleeping)
-		M.sleeping--
```

Comment:
https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/reagents/Chemistry-Reagents.dm#L773

Our induced comas last only for a single cycle and are paralysis based, not sleeping, so this doesn't really accomplish anything---even if it did, it wouldn't really make them wake up any faster.